sdk/trace/trace_test.go

```diff
@@ -273,14 +270,8 @@ func TestSetSpanAttributesOverLimit(t *testing.T) {
 		ParentSpanID: sid,
 		Name:         "SpanAttributesOverLimit/span0",
 		Attributes: []core.KeyValue{
-			{
-				Key:   core.Key("key1"),
-				Value: core.Value{Type: core.STRING, String: "value3"},
-			},
-			{
-				Key:   core.Key("key4"),
-				Value: core.Value{Type: core.STRING, String: "value4"},
-			},
+			key.String("key1", "value3"),
+			key.String("key4", "value4"),
```

Comment:
can you please add non-string attributes here ([comment](https://github.com/open-telemetry/opentelemetry-go/pull/143/files/1d02a09f0171d9e3e2f066bf658e91c89c028dd0#r328368123) in #143)?

Reply:
done